viewer/packages/pointcloud-styling/src/StyledPointCloudObjectCollection.ts

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import { PointCloudObjectCollection } from './PointCloudObjectCollection';
+import { CompletePointCloudAppearance } from './PointCloudAppearance';
+
+/**
+ * Represents an object collection that is associated with an appearance
+ */
+export class StyledPointCloudObjectCollection {
+  constructor(public objectCollection: PointCloudObjectCollection, public style: CompletePointCloudAppearance) {}
+}
```

Comment:
I guess not.. In `Cognite3DModel` we just type this out directly everywhere instead of having a separate class. I thought it is used many enough times that it's worth having its own class, or at least type. But it's not a big deal to revert if you disagree